sql/plan/indexed_table_access.go

```diff
@@ -237,15 +237,18 @@ func (i *IndexedTableAccess) String() string {
 	if !i.lookup.IsEmpty() {
 		children = append(children, fmt.Sprintf("filters: %s", i.lookup.Ranges.DebugString()))
 	}
+
 	if pt, ok := i.Table.(sql.ProjectedTable); ok {
-		var columns []string
-		for _, c := range pt.Projections() {
-			columns = append(columns, strings.ToLower(c))
-		}
-		if len(pt.Projections()) > 0 {
+		projections := pt.Projections()
```

Comment:
i prefer explicitly listing the cols, otherwise i have to go hunt down the schema to know whether index positions are off in a plan